src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -18,6 +18,20 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.Environment"/>
         public const string Environment = "DD_ENV";
 
+        /// <summary>
+        /// Configuration key for the application's version. Sets the "version" tag on every <see cref="Span"/>.
+        /// </summary>
+        /// <seealso cref="TracerSettings.Version"/>
+        public const string Version = "DD_VERSION";
+
+        /// <summary>
+        /// Configuration key for the application's default service name.
+        /// Used as the service name for top-level spans,
+        /// and used to determine service name of some child spans.
+        /// </summary>
+        /// <seealso cref="TracerSettings.ServiceName"/>
```

Comment:
Wasn't sure on wording for overriding `DD_SERVICE_NAME`. Thoughts?

Reply:
In other cases where we have renamed a setting we only have the official value (`DD_SERVICE`) here and the fallbacks (`DD_SERVICE_NAME`) are in `TracerSettings`. I think I would do the same here for consistency.

Long term, we should probably refactor these to be arrays of strings so we can have all the fallbacks right here and we would try them in order. Something like:
```csharp
public static readonly string[] Service = { "DD_SERVICE", "DD_SERVICE_NAME"};
```

Reply:
Done in 1c1b222185b9dbeacf6849d99d8d5181dfe888c9. Later we can revisit the array of fallbacks